playground/Qdrant/Qdrant.ApiService/Program.cs

```diff
@@ -3,7 +3,7 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
-// Add service defaults & Aspire components.
```

Comment:
Is this right though? cc @DamianEdwards. Service Defaults won't really add Aspire Integrations to your services, it does add ServiceDiscovery which will help in finding those, but doesn't seem like the comment was accurate here unless "Components" here really meant the "Aspire things"

Reply:
The comment should read `// Add service defaults & Aspire client integrations.` now I think. The comment mentions both because the intent was this is where you call methods that extend `IHostApplicationBuilder` directly, which is what service defaults and Aspire client integrations do.

Reply:
> which is what service defaults and Aspire client integrations do.

Yep, this was my understanding of the comments intent.

But good point that we should be specific about these being "client integrations" so users can discern what parts of the docs they should be reading when adding code here.